infrastructure/terraform/components/reporting/variables.tf

```diff
@@ -79,3 +79,9 @@ variable "core_env" {
   description = "The core environment that contains the corresponding Glue table/S3 buckets etc."
   default     = "internal-dev"
 }
+
+variable "log_retention_days" {
+  type        = number
+  description = "How many days to retain the logs generated by the step function"
+  default     = 60
```

Comment:
im sure ive seen the default for CW logs be 30 elsewhere in this repo due to Splunk forwarding, is this ok @sidnhs

Reply:
it looked to be set to 60 in most cases across the comms-mgr repo so I went with that.

Reply:
iirc the lambda logs are 30 and api gw logs are 60 by default on core

Reply:
ah ok, I'll amend to 30 to be in line with the lambda logs as they share a lot in common.